### PR TITLE
ENH: add a documentation link to the module page to bot settings

### PR DESCRIPTION
**Pull request by ghost**
This commit add a link to the bot settings page on intelmq.readthedocs.org with an anchor to the specific module.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
rst don't allow anchors with dots, they get converted to hyphens `#intelmq-bots-collectors-amqp-collector-amqp` automatically.

And please add a changelog entry =)

(Haven't tested the code yet)

**Comment by ghost** [comment]
> rst don't allow anchors with dots, they get converted to hyphens `#intelmq-bots-collectors-amqp-collector-amqp` automatically.

And underscores need to be replaced with hyphens as well :/

**Comment by ghost** [comment]
> rst don't allow anchors with dots, they get converted to hyphens `#intelmq-bots-collectors-amqp-collector-amqp` automatically.
> 
> And please add a changelog entry =)

Fixed.

File: intelmq_manager/static/js/configs.js [hunk 1]
```diff
@@ -373,6 +375,9 @@ function fill_bot(id, group, name) {
         insertKeyValue(key, bot.defaults[key], BORDER_TYPES.DEFAULT, false);
     }
 
+    var periodpos = bot['module'].lastIndexOf('.');
```
Comment (e3rd) [inline intelmq_manager/static/js/configs.js]: Might be `const` instead of `var`. I know I know the code is full of `var` but that's an old way.

**Review by e3rd — APPROVED** [review]
Seems useful!

**Pull request merged**